tsconfig.esm.json

```diff
@@ -2,6 +2,6 @@
   "extends": "./tsconfig",
   "compilerOptions": {
     "outDir": "dist/esm",
-    "module": "es6"
+    "module": "esnext"
```

Comment:
This is necessary to support [dynamic imports](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import#dynamic_imports) for the configuration file.